pkgs/applications/networking/p2p/yoink/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildGoModule
+}:
+
+let
+  version = "0.5.0";
+in
+buildGoModule {
+  pname = "yoink";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "MrMarble";
+    repo = "yoink";
+    rev = "v${version}";
+    hash = "sha256-9ftlAECywF4khH279h2qcSvKRDQX2I7GDQ7EYcEybL0=";
+  };
+
+  vendorSha256 = "sha256-cnfh2D/k4JP9BNlI+6FVLBFyk5XMIYG/DotUdAZaY0Q=";  
+  meta = with lib; {
```

```suggestion
  meta = {
```

tacaking issue: #292468